lib/tasks.js

```diff
@@ -60,6 +62,18 @@ exports.packageApp = {
   }
 }
 
+exports.saveApp = {
+  title: 'Save packaged app',
+  enabled: (ctx) => Boolean(ctx.args['--output']),
+  task: async (ctx, task) => {
+    const { zipFile } = ctx
+    if (!zipFile) throw new Error('Missing ctx.zipFile')
+    const path = resolvePath(resolvePath(ctx.args['--output']))
```

Comment:
Why are you calling `resolvePath` twice? 🤔

Reply:
wops 😂